### ISSUE TITLE
Request: Implement item module

### ISSUE THREAD

**Issue by ljvdhooft** [ansible-collections/community.zabbix#1203]
##### SUMMARY
Just like the commands to create a host, being able to create an item for the host.

##### COMPONENT NAME
community.zabbix.zabbix_item

##### ADDITIONAL INFORMATION
Use it to create items on a host to make provisioning easier. Right now this could be done via the HTTP API but it's pretty complicated since there are no checks if the items already exists, so the tasksfile gets quite big.

```yaml

- community.zabbix.zabbix_item:
  item_name: item1
  host_name: parent_host
  state: present
  type: http_agent
  url: "google.com"
  preprocessing: [
  { -- add some params -- }
  ]
  
```

**Comment by ljvdhooft** [comment]
proof of concept:
https://github.com/ljvdhooft/community.zabbix/commit/4e584a5e9ae2997ab4e54bf160ad2aca69e5ee73

A lot of parameters are missing, but the basics are working.

**Comment by ljvdhooft** [comment]
With the other parameters and documentation added:
https://github.com/ljvdhooft/community.zabbix/commit/046ed956f7d4157b96c89ecf390330026ee66a9d

Still examples need to be made and will begin on integration tests

**Comment by pyrodie18** [comment]
This was implemented in #1222

**Comment by ljvdhooft** [comment]
Hi, unfortunately this doesn't support parameter idempotency like for example zabbix_host. I didn't expect my PR to raise errors during tests since tests completed successfully before committing. Perhaps I'll take a look at the module and create a new PR based on the now merged code
